app/helpers/admin_helper.rb

```diff
@@ -2,4 +2,11 @@ module AdminHelper
   def support_email_address
     'report-mi@crowncommercial.gov.uk'
   end
+
+  def link_to_suppliers(suppliers)
+    supplier_links = suppliers.collect do |supplier|
+      link_to h(supplier.name), admin_supplier_path(supplier)
```

Comment:
The `h` helper method is unnecessary, since Rails now escapes strings on display by default.

Reply:
Ah okay, this was pointed out to me as a security precaution, do you know which version of Rails it became unnecessary in?